systemd/coreos-installer-service

```diff
@@ -92,6 +92,10 @@ fi
 # Ensure device nodes have been created
 udevadm settle
 
+# Enable the service for reporting failures to console
+echo "systemctl enable coreos-installer-emergency.service"
```

Comment:
This does not seem right, either this or enabling the service as a `postprocess` in `manifest.yaml` under FCOS config directory would work, though it will be much better if the service is enabled by default..

Reply:
We should do this from the generator.